File: 프로그래머스/0/181852. 뒤에서 5등 위로/뒤에서 5등 위로.cs
using System;

public class Solution {
    public int[] solution(int[] num_list) {
        
        int[] a = new int[num_list.Length - 5];
        Array.Sort(num_list);
        
        for (int i = 5; i < num_list.Length; i++)
        {
            a[i - 5] = num_list[i];

        }
        
        return a;
    }
}

File: 프로그래머스/0/181852. 뒤에서 5등 위로/뒤에서 5등 위로.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

vector<int> solution(vector<int> num_list) {
    vector<int> answer;
    sort(num_list.begin(), num_list.end());
    num_list.erase(num_list.begin(), num_list.begin()+5);
  
    return num_list;
}